watercore-base/src/main/java/me/lucyy/watercore/core/WaterCorePlugin.java

```diff
@@ -1,49 +1,28 @@
-/*
- * Copyright © 2021 Lucy Poulton.
- * This file is part of watercore.
- *
- * watercore is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * watercore is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with watercore.  If not, see <https://www.gnu.org/licenses/>.
- */
-
 package me.lucyy.watercore.core;
 
-import me.lucyy.common.format.Platform;
-import me.lucyy.watercore.api.WaterCore;
 import me.lucyy.watercore.api.impl.WaterCoreImpl;
-import me.lucyy.watercore.modules.core.CoreModule;
+import me.lucyy.watercore.core.mysql.MySqlHandler;
+import org.bukkit.configuration.file.FileConfiguration;
 import org.bukkit.plugin.java.JavaPlugin;
 
-@SuppressWarnings("unused")
 public final class WaterCorePlugin extends JavaPlugin {
+	private WaterCoreImpl waterCore;
+	public static FileConfiguration config;
```

Comment:
Please get rid of this static. Start learning to use getters/setters.

Reply:
Several things:
1. MemoryDataStore, FormatProvider, 
2. Whats wrong with concatenation? (no seriously?
3. getConfig() is not static, so getters and setters no work
4. I will work on implementing DataStore and key-value stuff in the future, this is just basic for now
5. Don't worry I wont delete your checkstyle, or anything I didn't write

Reply:
In regard to 5, you have almost 1500 line deletions, you're still behind main